chess_api/chess_api/chess_api.py

```diff
@@ -0,0 +1,56 @@
+from flask import (
+    Blueprint, request, jsonify
+)
+from . import chess_logic
+
+bp = Blueprint('chess_api', __name__, url_prefix='/chess_api')
+
+@bp.route("/start-game", methods=["POST"])
+def initialize_board():
+    if request.method == "POST":
+        chess_logic.initialize_board()
+        try:
+            return "Chess engine set and board cleared."
+        except:
+            return "Could not determine intent"
+
+@bp.route("/get-best-move", methods=["GET"])
+def get_best_move():
+    if request.method == "GET":
+        best_move = chess_logic.get_best_move()
+
+        try:
+            return jsonify(best_move)
+        except:
+            return "Could not determine intent"
+
+@bp.route("/get-current-player", methods=["GET"])
+def get_current_player():
+    if request.method == "GET":
+        current_player = chess_logic.get_current_player()
+
+        try:
+            return jsonify(current_player)
```

Comment:
`jsonify` should only accept a python dictionary as a parameter. It looks like the values of `current_player` are limited to `"WHITE"` or `"BLACK"`. So, I would either remove `jsonify` or convert it into a dictionary, like `{"currPlayer": current_player}`. Double-check the other `jsonify` calls as well.